dev/scripts/src/alluxio.org/build-distribution/cmd/common.go

```diff
@@ -63,6 +63,25 @@ var ufsModules = map[string]module{
 	"ufs-hadoop-3.3": {"hadoop-3.3", false, "-pl underfs/hdfs -Pufs-hadoop-3 -Dufs.hadoop.version=3.3.0 -PhdfsActiveSync"},
 }
 
+var libJars = map[string]struct{}{
+	"integration-tools-hms":        {},
```

Comment:
i just noticed that the list of jars names here do not match with the list of jars removed from generate-tarball.go

this new list is missing ozone, cephfs, and cephfs-hadoop but adds kodo. is this change intentional? i haven't taken a look at the list of published jars for a while so my knowledge may be out of date

Reply:
Sorry, my fault. I think I just missed `ozone, cephfs, cephfs-hadoop`. 

As far as `kodo`, it is a new underfs? I don't know. I notice that `kodo` is included in the `underfs/pom.xml`, so it is supposed to be included in a release version. But `kodo` is missed in the original generate-tarball.go. So I added it to the libjar names.

Reply:
for this PR, can we keep all the original ufs jars and not make any modifications to the list? i don't think there is any issue in adding the kodo jar, but i would like to get the opinion of other people, so it would be better suited as a separate PR

Reply:
Of course. I'll fix it back.